packages/block-library/src/post-featured-image/block.json

```diff
@@ -86,6 +89,9 @@
 			"clientNavigation": true
 		}
 	},
+	"selectors": {
+		"shadow": ".wp-block-post-featured-image img, .wp-block-post-featured-image .wp-block-image__crop-area, .wp-block-post-featured-image .components-placeholder"
+	},
```

Comment:
I don't think the Post Featured Image block has a cropper like the image block. So the cropper related selectors here are probably able to be dropped.